### PR TITLE
TTAHUB-86 Hide the Report Approved Section when Printing

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

The purpose of this JIRA is to hide the Reports Approved section along with both Creator / Manager notes.

Per JIRA comments these sections should NEVER be contained in the print out.

## How to test

Login to the application and view a Approved report as BOTH the Submitter and Approver. When you go to print the AR via the browser print both notes sections should be missing in the print out.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-86

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Approved.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const Approved = ({
   const managerNotesState = getEditorState(managerNotes || 'No manager notes');
 
   return (
-    <>
+    <div className="smart-hub--hide-notes">
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/ActivityReport/Pages/Review/Approver/Approved.js:16]: Not sure if there is a cleaner way to add this class. Adding the class name to the Fragment didn't work.

File: frontend/src/pages/ActivityReport/Pages/Review/Submitter/Approved.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const Approved = ({
   const managerNotesState = getEditorState(managerNotes || 'No manager notes');
 
   return (
-    <>
+    <div className="smart-hub--hide-notes">
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/ActivityReport/Pages/Review/Submitter/Approved.js:16]: Not sure if there is a cleaner way to add this class. Adding the class name to the Fragment didn't work.

**Review by thewatermethod — COMMENTED** [review]
One thing that is available to us (that I only recently discovered myself) is the "no-print" css class. I would recommend removing the "smart-hub--hide-notes" from the markup and css and reusing that "no-print" class instead.

**Comment by AdamAdHocTeam** [comment]
> One thing that is available to us (that I only recently discovered myself) is the "no-print" css class. I would recommend removing the "smart-hub--hide-notes" from the markup and css and reusing that "no-print" class instead.

I have updated the code based on this comment can you take a look again when you have a moment.

**Review by thewatermethod — APPROVED** [review]
Looks good, thanks for making that change!

**Pull request merged**